cogs/owner.py

```diff
@@ -554,6 +554,38 @@ def check(m):
         else:
             await self.bot.say("User is not blacklisted.")
 
+    @blacklist.command(name="hackadd", pass_context=True)
+    async def _blacklist_hackadd(self, ctx, user):
+        """Adds user to Red's global blacklist using its ID."""
+
+        test = discord.utils.get(ctx.message.server.members, id=user)
+        if test is None:
+            await self.bot.say("User not found. Are you sure you provided the good user ID?")
+            return
+        else:
+            if user not in self.global_ignores["blacklist"]:
+                self.global_ignores["blacklist"].append(user)
+                self.save_global_ignores()
+                await self.bot.say("User has been blacklisted.")
+            else:
+                await self.bot.say("User is already blacklisted.")
+
+    @blacklist.command(name="hackremove", pass_context=True)
+    async def _blacklist_hackremove(self, ctx, user):
+        """Remove user to Red's global blacklist using its ID."""
+
+        test = discord.utils.get(ctx.message.server.members, id=user)
+        if test is None:
+            await self.bot.say("User not found. Are you sure you provided the good user ID?")
+            return
+        else:
+            if user not in self.global_ignores["blacklist"]:
```

Comment:
Uhh, why would you want to remove a user that's NOT in the blacklist? I'd drop the `not` on this line and your issue should be fixed